examples/index.json

```diff
@@ -1201,6 +1201,13 @@
         "verifyBuild": true,
         "verifyOutput": true,
         "isLearnByExample": true
```

Comment:
Should we rename the BBE before this?

Reply:
@sm1990 do we need to do anything for redirects with this change in BBE name?

Reply:
@poorna2152 note that the regexp type BBE has a link to this too.

Reply:
@MaryamZi yes. I'll have to update the Google search index. 
@poorna2152 Please share the old BBE url and the new BBE url.

Reply:
Old link: https://ballerina.io/learn/by-example/regexp-operations/
New link: https://ballerina.io/learn/by-example/regexp_operations_overview/